.eslintrc.js

```diff
@@ -1,7 +1,8 @@
 module.exports = {
   env: {
     browser: true,
-    es6: true
+    es6: true,
+    "jest/globals": true
```

Comment:
This way eslint will know that global variables defined by jest will exist even when they're not required.